metadata-ingestion/docs/sources/dynamodb/dynamodb_recipe.yml

```diff
@@ -0,0 +1,25 @@
+source:
+  type: dynamodb
+  config:
+    platform_instance: "AWS_ACCOUNT_ID"
+    aws_access_key_id: "${AWS_ACCESS_KEY_ID}"
+    aws_secret_access_key: "${AWS_SECRET_ACCESS_KEY}"
+    # User could use the below option to provide a list of primary keys of a table in dynamodb format,
+    # those items from given primary keys will be included when we scan the table.
+    # For each table we can retrieve up to 16 MB of data, which can contain as many as 100 items.
+    # We'll enforce the the primary keys list size not to exceed 100
+    # The total items we'll try to retrieve in these two scenarios:
+    # 1. If user don't specify include_table_item: we'll retrieve up to 100 items
+    # 2. If user specifies include_table_item: we'll retrieve up to 100 items plus user specified items in
+    # the table, with a total not more than 200 items
+    # include_table_item:
+    #   table_name:
+    #     [
```

Comment:
Sorry, this is not clear to me in spite of the description. Can you share an example of how this will look in real ?

Reply:
For example if I have a table `Reply` and I have a key `Id` and partition key `ReplyDateTime`, there are two items I would like to include when scanning the table because these two items might have most representative fields of this table, this is the config I will put in `include_table_item`:
```
    include_table_item:
      Reply:
        [
          {
            "ReplyDateTime": { "S": "2015-09-22T19:58:22.947Z" },
            "Id": { "S": "Amazon DynamoDB#DynamoDB Thread 1" },
          },
          {
            "ReplyDateTime": { "S": "2015-10-05T19:58:22.947Z" },
            "Id": { "S": "Amazon DynamoDB#DynamoDB Thread 2" },
          },
        ]
```
This is the input format that will be used by `dynamodb_client.batch_get_item()`, but it seems if we put this example in the sample recipe might be too much that confused users.

Reply:
Oh, it makes sense now. I believe this would be more intuitive to a DyanamoDB user. Could you please add a section "Advanced Configurations" in `dynamodb_post.md` with this description and example ?
Similar to [this](https://datahubproject.io/docs/generated/ingestion/sources/kafka-connect/#advanced-configurations) .

Reply:
Thank you! I've updated the `dynamodb_post.md` to include this example and update `Limitations` accordingly, also added a `TODO` for the previous comment.